src/xbps/repositories/mirrors/index.md

```diff
@@ -53,3 +53,12 @@ sub-repository.
 
 Void Linux is also mirrored on the Tor network. See [Using Tor
 Mirrors](./tor.md) for more information.
+
+## Creating a mirror
+
+If you'd like to set up a mirror, follow one of the many guides available (e.g.
+[this EFF
+guide](https://www.eff.org/keeping-your-site-alive/mirroring-your-site)), then
+submit a pull request to [the void-docs
+repository](https://github.com/void-linux/void-docs) to add your mirror to this
+page. A mirror requires at least 1.1TB of storage space.
```

Comment:
"a full mirror requires around 1TB of storage", to allow for increases. Does adding "full" there help with making it clear that the mirror can have less than all the repositories?

Reply:
Oh, well, i thought that "at least" covered the possibility of increases. :-) But i'll change it. And yes, adding "full" is good, i think. PR updated.